src/types/BeachballOptions.ts

```diff
@@ -30,6 +30,7 @@ export interface CliOptions
   all: boolean;
   authType: AuthType;
   bump: boolean;
+  bumpPeerDeps?: boolean;
```

Comment:
Can you move this to the `Pick<RepoOptions, ...>` list above?

Reply:
Sure moved.